experience.css

```diff
@@ -0,0 +1,108 @@
+* {
+  font-family: Impact, fantasy;
+  font-size: 20px;
+}
+
+body {
+  background-color: lightgray;
+}
+
+nav {
```

Comment:
consider having a stylesheet that includes shared styles from all pages, as well as individual stylesheets for specific pages' styles. that way you don't have to repeat any lines of code